plugins/accessibility/index.js

```diff
@@ -1,14 +1,14 @@
 var q = require('q'),
     fs = require('fs'),
     path = require('path'),
-    _ = require('lodash');
+    request = require('request'),
+    Entities = require('html-entities').XmlEntities;
 
 /**
- * You can enable this plugin in your config file:
- *
- *    // The Chrome Accessibility Developer Tools are currently
- *    // the only integration option.
+ * You can audit your website against the Chrome Accessibility Developer Tools,
+ * Tenon.io, or both by enabling this plugin in your config file:
```

Comment:
I put it just below in the Tenon section, and the name itself is a URL. Since the Chrome version wasn't linked, I figured we were already covered.